frontend/src/app/components/Visibility/index.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+import React from 'react';
+import classnames from 'classnames';
+
+type BannerProps = {
+  isVisible?: boolean,
+  className?: string,
+  children?: Array<any>
+};
+
+type BannerState = {
+  isVisible: boolean
+};
+
+export default class Visibility extends React.Component {
```

Comment:
Also "Visibility" might be too general a name for this utility component, since so far it only checks to see if the element is vertically higher than current scroll position. Doesn't do the inverse - which could be handy for simplifying the sticky header on experiment pages. Figured that could be for a future enhancement though